open-company/open-company-web
# Fix unread posts for anonymous users

Bug:
- create a public section
- add a new post
- add another post
- add a comment to the second post
- sign in another org
- open the section URL (not the post URL)
- the last added post has white bg

To test:
- with user 1 of org A 
- create section S with public access
- add a new post
- add another post
- with user 2 of org A
- open AP
- see 2 white backgrounded posts?
- open both to mark them read
- go back to AP
- with user 1 add a comment
- user 2 sees the post with white background again?
- now with user 3 from org B
- open section S of org A
- both posts have grey background?
